support/nginx/peertube

```diff
@@ -1,202 +1,239 @@
+# Minimum Nginx version required:  1.13.0 (released Apr 25, 2017)
+
 # Uncomment in production to redirect HTTP to HTTPS. Leave commented for docker-compose.
 #server {
 #  listen 80;
 #  listen [::]:80;
 #  server_name ${WEBSERVER_HOST};
-#
-#  location /.well-known/acme-challenge/ {
```

Comment:
I think we need this block when we fetch our first cert if we don't use the standalone webserver of certbot

Reply:
As mentioned in the PR description, Let’s Encrypt supports 301 redirects, so this block is superfluous.

Reply:
Yes but it will be redirected on the https version, but if it's for the first cert it will fail no?

Reply:
Admittedly I didn’t test that part’s impact. Scaling back, I don’t want to break things.